addon/helpers/page-title.js

```diff
@@ -40,7 +59,7 @@ export default Helper.extend({
     hash.id = guidFor(this);
     hash.title = params.join('');
     tokens.push(hash);
-    scheduleOnce('afterRender', get(this, 'headData'), updateTitle, tokens);
+    scheduleOnce('afterRender', updateTitle, get(this, 'document'), tokens, get(this, 'isFastBoot'));
```

Comment:
Instead of passing in `isFastBoot` to the function, lets have the function directly do:

```js
let isFastBoot = typeof FastBoot !== 'undefined';
```